src/tests/test_connection.py

```diff
@@ -583,7 +583,10 @@ def test_feedback_when_switching_connection_with_descriptors(
     ip_empty.run_cell("%sql sqlite://")
 
     captured = capsys.readouterr()
-    assert "Switching to connection sqlite://" == captured.out.splitlines()[-1]
+    assert (
+        "Connecting and Switching to Connection sqlite://"
```

Comment:
Switching shouldn't be capitalized

Reply:
(except when it is the first word in the sentence)

Reply:
[changed!](https://github.com/ploomber/jupysql/pull/796/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R230-R242)